cli/cmd/generate_terraform_test.go

```diff
@@ -0,0 +1,215 @@
+			toggleNonInteractive()
+			defer toggleNonInteractive()
+			dir, err := ioutil.TempDir("", "tf-install-test")
+			if err != nil {
+				panic(err)
+			}
+			defer os.RemoveAll(dir)
+
+			// Install some version that should be detected
+			installer := &releases.ExactVersion{
+				Product: product.Terraform,
+				Version: version.Must(version.NewVersion("0.12.31")),
+			}
+
+			installPath, err := installer.Install(context.Background())
+			if err != nil {
+				panic(err)
+			}
+			installedTempPath := filepath.Dir(installPath)
+			defer os.RemoveAll(installedTempPath)
+
+			// Update path to find the new version
+			oldPath := os.Getenv("PATH")
+			os.Setenv("PATH", fmt.Sprintf("%s:%s", installedTempPath, oldPath))
+			defer os.Setenv("PATH", oldPath)
+
+			// Locate or install should find version 0.12.31, determine it's too old, install newer version
+			tf, err := LocateOrInstallTerraform(false, dir)
+			if err != nil {
+				panic(err)
+			}
+			defer os.RemoveAll(filepath.Dir(tf.ExecPath()))
+			version, _, _ := tf.Version(context.Background(), true)
+
+			assert.NoError(t, err)
+			assert.Equal(t, installTerraformVersion, version.String())
+			assert.NotEqual(t, installedTempPath, filepath.Dir(tf.ExecPath())) // We shouldn't be using the old version we installed
+		})
+	}
+}
+
+// Test the feedback given to user after running terraform plan or apply
+func TestGenerationTfExecutionFeedback(t *testing.T) {
+	t.Run("init not run", func(t *testing.T) {
+		out := provideGuidanceAfterExit(false, false, "/place", "/thing/terraform")
+		assert.Equal(t,
+			`Terraform code saved in /place
+
+The generated code can be executed at any point in the future using the following commands:
+  cd /place
+  /thing/terraform init
+  /thing/terraform plan
+  /thing/terraform apply
+
+`, out)
+	})
+	t.Run("init run but no plan executed", func(t *testing.T) {
+		out := provideGuidanceAfterExit(true, false, "/place", "/thing/terraform")
+		assert.Equal(t,
+			`Terraform code saved in /place
+
+The generated code can be executed at any point in the future using the following commands:
+  cd /place
+  /thing/terraform plan
+  /thing/terraform apply
+
+`, out)
+	})
+	t.Run("after plan but exit without running apply", func(t *testing.T) {
+		out := provideGuidanceAfterExit(true, true, "/place", "/thing/terraform")
+		assert.Equal(t,
+			`Terraform code and plan output saved in /place
+
+The generated code can be executed at any point in the future using the following commands:
+  cd /place
+  /thing/terraform plan
+  /thing/terraform apply
+
+`, out)
+	})
+}
+
+// Test the feedback given after running terraform to completion (success)
+func TestGenerationTfExecutionSuccess(t *testing.T) {
+	t.Run("using default lacework profile", func(t *testing.T) {
+		out := provideGuidanceAfterSuccess("/tmp", "")
+		assert.Equal(t, `Lacework integration was successful! Terraform code saved in /tmp
+
+Use the Lacework CLI to view integration status:
+  lacework integration list
```

Comment:
nit: what about this format that we use in help messages and other guidance output:
```suggestion
To view integration status:
    lacework integration list
```

Reply:
👍